dwave/plugins/sklearn/nearest_neighbors.pyx

```diff
@@ -0,0 +1,117 @@
+# distutils: language = c++
+import numpy as np
+cimport numpy as np
+from libcpp.vector cimport vector
+
+cdef extern from "math.h":
+    double log(double x) nogil
```

Comment:
```suggestion
from libc.math cimport log
```

You can see it [here](https://github.com/cython/cython/blob/a174dcd02f1f662b61f66953929709fbb47b605e/Cython/Includes/libc/math.pxd#L120). If you're wondering how to find what Cython actually has already implemented... the answer is that I literally search the source files every. single. time. :smile: